flow_screen_components/richTextInputFSC/README.md

```diff
@@ -1,3 +1,45 @@
 # Rich Text Edit Control #
 
 https://unofficialsf.com/rich-text-input-flow-screen-component/
+
+
+# inputRichTextFSC
+Rich Text enhancements for Flows
+
+### Summary:
+IMPORTANT:  Use richerText = true input attribute if you want to leverage these enhanced features.  Otherwise leave empty and regular input Rich Text component will be used.
```

Comment:
I would call this attribute something like EnableAdvancedFeatures

Reply:
Great, thanks.  This will be in next update.  enableAdvancedTools